IaC/AKS/README.md

```diff
@@ -0,0 +1,373 @@
+export ARCH=linux-amd64
+curl -sL "https://github.com/istio/istio/releases/download/$ISTIO_VERSION/istioctl-$ISTIO_VERSION-$ARCH.tar.gz" | tar xz
+
+```
+
+Copy the istioctl client binary to the standard user program location in your PATH
+
+```bash
+
+sudo mv ./istioctl /usr/local/bin/istioctl
+sudo chmod +x /usr/local/bin/istioctl
+
+```
+
+Install the Istio Operator and Components on AKS
+
+```bash
+
+istioctl operator init
+kubectl create ns istio-system
+kubectl apply -f $REPO_ROOT/IaC/AKS/cluster/manifests/istio/istio.aks.yaml
+
+# the istio resources will take about a minute to be installed
+
+```
+
+Validate the Istio installation
+
+```bash
+
+kubectl get all -n istio-system
+
+```
+
+You should see the following components:
+
+- `istio*` - the Istio components
+- `jaeger-*`, `tracing`, and `zipkin` - tracing addon
+- `prometheus` - metrics addon
+- `grafana` - analytics and monitoring dashboard addon
+- `kiali` - service mesh dashboard addon
+
+Enable automatic sidecar injection in the default namespace:
+
+```bash
+
+kubectl label namespace default istio-injection=enabled
+
+```
+
+Get the public IP of the Istio Ingress Gateway and set the application endpoint.
+
+```bash
+
+export INGRESS_PIP=$(kubectl --namespace istio-system  get svc -l istio=ingressgateway -o jsonpath='{.items[0].status.loadBalancer.ingress[0].ip}')
+export He_App_Endpoint=http://${INGRESS_PIP}.nip.io
+
+```
+
+## Install KEDA
+
+KEDA autoscales the NGSA pods by assessing metrics for incoming requests, which are captured by Istio and stored in Prometheus.
+
+```bash
+
+kubectl create ns keda
+helm install keda kedacore/keda -n keda
+
+```
+
+## Deploy NGSA Secrets
+
+```bash
+
+kubectl create secret generic ngsa-secrets \
+  --from-literal=CosmosDatabase=$Imdb_DB \
+  --from-literal=CosmosCollection=$Imdb_Col \
+  --from-literal=CosmosKey=$(az cosmosdb keys list -n $Imdb_Name -g $Imdb_RG --query primaryReadonlyMasterKey -o tsv) \
+  --from-literal=CosmosUrl=https://${Imdb_Name}.documents.azure.com:443/ \
+  --from-literal=AppInsightsKey=$(az monitor app-insights component show -g $He_App_RG -a $He_Name --query instrumentationKey -o tsv)
+
+```
+
+
+
+## Deploy NGSA with Helm
+
+An helm chart is included for the reference application ([NGSA](https://github.com/retaildevcrews/ngsa))
+
+Install the Helm Chart located in the cloned directory
+
+```bash
+
+cd $REPO_ROOT/IaC/AKS/cluster/charts
+
+```
+
+```bash
+# Before installing the chart, ensure <INGRESS_PIP> is updated in your Values.yaml file with the value of $INGRESS_PIP
+
+# Install NGSA using the upstream ngsa image from Dockerhub
+helm install ngsa-aks ngsa/
+
+# the application generally takes about 2-4 minutes to be ready
+
+# check the version endpoint
+# you may get a timeout error, if so, just retry
+
+http ${He_App_Endpoint}/version
+
+```
+
+Run the Validation Test
+
+> For more information on the validation test tool, see [Web Validate](https://github.com/retaildevcrews/webvalidate)
+
```

Comment:
Would be good to add in the docker command to run webv. For the baseurl, you can use ngsa's Testfiles directory

Reply:
Discussed with Joaquin and decided it's better to add this in for issue #58